pages/settings/pats.vue

```diff
@@ -2,25 +2,33 @@
   <div class="universal-card">
     <ConfirmModal
       ref="modal_confirm"
-      title="Are you sure you want to delete this token?"
-      description="This will remove this token forever (like really forever)."
-      proceed-label="Delete this token"
+      :title="formatMessage(messages.deletePatModalTitle)"
+      :description="formatMessage(messages.deletePatModalDescription)"
+      :proceed-label="formatMessage(messages.deletePatModalProceedLabel)"
       @proceed="removePat(deletePatIndex)"
     />
     <Modal
       ref="patModal"
-      :header="`${editPatIndex !== null ? 'Edit' : 'Create'} personal access token`"
+      :header="
+        editPatIndex !== null
+          ? formatMessage(messages.editPatModalTitle)
+          : formatMessage(messages.createPatModalTitle)
+      "
     >
       <div class="universal-modal">
-        <label for="pat-name"><span class="label__title">Name</span> </label>
+        <label for="pat-name">
+          <span class="label__title">{{ formatMessage(commonMessages.nameLabel) }}</span>
```

Comment:
This probably shouldn't be a common message, at least not for now. ‘Name’ can mean different things depending on the context.

Reply:
Fixed ^^.